src/linux/backend-v4l2.cpp

```diff
@@ -784,7 +785,7 @@ namespace librealsense
             // are not available via mipi
             // TODO - find a way to assign unique id for mipi
             // maybe using bus_info and card params (see above in this method)
-            //info.unique_id = busnum + "-" + devpath + "-" + devnum;
+            info.unique_id = bus_info; // use bus_info as per camera unique id for mipi
```

Comment:
Seems to be enough, but could you please add here how would the busnum look like for each of the mipi devices, for example if 2 devices are connected?
Besides that, if this is chosen as unique_id, the above comment can be removed.

Reply:
Thank you.
Below is an example of busnum as well as the card param: [bus_info -- card_info] of 4 x D457 mipi devices. Bus_info seems to be unique to each mipi device and work in current system. 

```
platform:15c10000.vi:0 -- vi-output, DS5 mux 34-0010
platform:15c10000.vi:0 -- 15c10000.vi-metadata-0
platform:15c10000.vi:0 -- vi-output, DS5 mux 35-0010
platform:15c10000.vi:0 -- 15c10000.vi-metadata-0
platform:15c10000.vi:0 -- vi-output, DS5 mux 36-0010
platform:15c10000.vi:0 -- vi-output, DS5 mux 37-0010

platform:15c10000.vi:2 -- vi-output, DS5 mux 38-0010
platform:15c10000.vi:2 -- 15c10000.vi-metadata-2
platform:15c10000.vi:2 -- vi-output, DS5 mux 39-0010
platform:15c10000.vi:2 -- 15c10000.vi-metadata-2
platform:15c10000.vi:2 -- vi-output, DS5 mux 40-0010
platform:15c10000.vi:2 -- vi-output, DS5 mux 41-0010

platform:15c10000.vi:4 -- vi-output, DS5 mux 42-0010
platform:15c10000.vi:4 -- 15c10000.vi-metadata-4
platform:15c10000.vi:4 -- vi-output, DS5 mux 43-0010
platform:15c10000.vi:4 -- 15c10000.vi-metadata-4
platform:15c10000.vi:4 -- vi-output, DS5 mux 44-0010
platform:15c10000.vi:4 -- vi-output, DS5 mux 45-0010

platform:15c10000.vi:5 -- vi-output, DS5 mux 46-0010
platform:15c10000.vi:5 -- 15c10000.vi-metadata-5
platform:15c10000.vi:5 -- vi-output, DS5 mux 47-0010
platform:15c10000.vi:5 -- 15c10000.vi-metadata-5
platform:15c10000.vi:5 -- vi-output, DS5 mux 48-0010
platform:15c10000.vi:5 -- vi-output, DS5 mux 49-0010
```

Reply:
Seems great. Thanks